src/bundle/Resources/config/default_parameters.yml

```diff
@@ -1,32 +1,32 @@
 parameters:
-    ezsettings.admin_group.content_edit.templates.create_draft: 'EzPlatformAdminUiBundle:content/content_edit:content_create_draft.html.twig'
-    ezsettings.admin_group.user_edit.templates.update: 'EzPlatformAdminUiBundle:content/content_edit:user_edit.html.twig'
-    ezsettings.admin_group.user_edit.templates.create: 'EzPlatformAdminUiBundle:content/content_edit:user_create.html.twig'
+    ezsettings.admin_group.content_edit.templates.create_draft: '@@ezdesign/content/content_edit/content_create_draft.html.twig'
```

Comment:
what is the story behind double @@?

Reply:
Symfony treats this as a service and moreover injecting services in parameters is not allowed, so it results in a parse error. Here I'm escaping `@`.